### DISCUSSION TITLE
md/java/basic

### DISCUSSION THREAD

**Discussion by giscus** [HelloCode66/Java-Navigation#2] [General]
# md/java/basic

::: tip 在开始学习前，请确保已正确配置了开发环境，Java官网：https://www.java.com/zh-CN/ ::: 一、基础语法 1、第一个Java程序 ```java public class Hello{									//定义一个名为Hello的类（ public static void main(String[] args...

http://localhost:8080/md/java/basic.html

**Comment by jiusi66** [comment]
- 评论测试
